enterprise/externalproxy/proxy.go

```diff
@@ -0,0 +1,184 @@
+package externalproxy
```

Comment:
I think calling this `workspaceproxy` would be better. `externalproxy` reminds me of `nginx`.

Reply:
Alternatively, we could do `wsproxy` and then have `wsproxysdk`. Then it's not super long either, and kinda fits the agent and agentsdk naming scheme.